### ISSUE TITLE
[Bug] Search or enter address does not have correct description

### ISSUE THREAD

**Issue by sblatz** [mozilla-mobile/fenix#7966]
## Steps to reproduce
1. Enable Talkback
2. Go from home screen to search screen by tapping "search or enter address"

### Expected behavior
1. Search screen opens and announces your default search engine followed by "search or enter address"

### Actual behavior
1. Search screen opens and announces your default search engine

### Device information

* Android device: ?
* Fenix version: ?

**Comment by eliserichards** [comment]
Adding `android:searchSettingsDescription="@string/search_hint">` to `searchable.xml` reads "search or enter address" but no longer reads the default search engine

**Comment by abodea** [comment]
Verified as fixed on the latest Nightly 7/20 with Google Pixel 4 XL (Android 11).
